src/page-templates/content-type/content-type.tsx

```diff
@@ -424,166 +141,183 @@ const ContentTypePage: NextPage<ContentTypePageProps> = ({
                 title={pageTitle}
             />
             <Hero
-                crumbs={[{ text: 'MongoDB Developer Center', url: '/' }]}
+                crumbs={heroCrumbs}
                 name={pluralize(contentType)}
                 description={description}
-                ctas={
-                    shouldRenderRequestButton(contentType) ? CTAElement : null
-                }
+                ctas={heroCTAs}
             />
+
+            {/* Main content body */}
             <div sx={pageWrapper}>
                 <GridLayout
                     sx={{
                         rowGap: 0,
                     }}
                 >
-                    <DesktopFilters
-                        sx={desktopFiltersStyles}
-                        onFilter={filters => {
-                            onFilter(filters);
-                            clearPagination();
-                        }}
-                        allFilters={allFilters}
-                        l1Items={l1Items}
-                        languageItems={languageItems}
-                        technologyItems={technologyItems}
-                        contributedByItems={contributedByItems}
-                        expertiseLevelItems={expertiseLevelItems}
-                        codeLevelItems={
-                            contentType === 'Code Example' ? codeLevelItems : []
-                        }
-                    />
                     <div
                         sx={{
-                            gridColumn: ['span 6', null, 'span 8', 'span 9'],
+                            gridColumn: 'span 3',
                         }}
                     >
-                        <Grid
-                            columns={[1, null, 3]}
-                            sx={searchBoxSortBarWrapperStyles}
-                        >
-                            <div
-                                sx={{
-                                    ...searchBoxStyles,
-                                    ...(!!searchString || hasFiltersSet
-                                        ? {}
-                                        : { gridColumn: 'span 3' }),
+                        <DesktopFilters
+                            {...filterProps}
+                            sx={desktopFiltersStyles}
+                            filterItems={filterItems}
+                        />
+
+                        {mobileFiltersOpen && (
+                            <MobileFilters
+                                {...filterProps}
+                                {...sortBoxProps} // Mobile filters include sorting
+                                filterItems={filterItems}
+                                closeModal={() => setMobileFiltersOpen(false)}
+                            />
+                        )}
+                    </div>
+
+                    {!children && (
+                        <div sx={searchWrapperStyles}>
```

Comment:
Using a flexbox here to deduplicate some markup (had to duplicate the sortbox before when we were using a grid layout based on if the featured section was showing)